terraform/auth/config.go

```diff
@@ -425,3 +426,18 @@ func (c *Config) SharedfilesystemV2Client(region string) (*gophercloud.ServiceCl
 func (c *Config) KeyManagerV1Client(region string) (*gophercloud.ServiceClient, error) {
 	return c.CommonServiceClientInit(openstack.NewKeyManagerV1, region, "key-manager")
 }
+
+// A wrapper to determine if logging in gophercloud should be enabled, with a fallback
+// to the OS_DEBUG environment variable when no explicit configuration is passed.
+func enableLogging(enable bool) bool {
```

Comment:
Figured I make this a _private_ utility function, e.g. not part of the config class. But let me know if that's wanted or if you prefer this differently.